post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-configuration.adoc

```diff
@@ -22,21 +22,56 @@ For information on migrating your single-architecture cluster to a cluster that
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-aws.adoc#creating-multi-arch-compute-nodes-aws[Creating a cluster with multi-architecture compute machines on AWS]
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-bare-metal.adoc#creating-multi-arch-compute-nodes-bare-metal[Creating a cluster with multi-architecture compute machines on bare metal]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-gcp.adoc#creating-multi-arch-compute-nodes-gcp[Creating a cluster with multi-architecture compute machines on GCP]
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc#creating-multi-arch-compute-nodes-ibm-z[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with z/VM]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-bare-metal.adoc#creating-multi-arch-compute-nodes-bare-metal[Creating a cluster with multi-architecture compute machines on bare metal]
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z-kvm.adoc#creating-multi-arch-compute-nodes-ibm-z-kvm[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with {op-system-base} KVM]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc#creating-multi-arch-compute-nodes-ibm-z[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with z/VM]
+|&#10003;
+|
+|`x86_64`, `s390x`
+|`x86_64`, `s390x`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc#creating-multi-arch-compute-nodes-ibm-power[Creating a cluster with multi-architecture compute machines on {ibm-power-name}]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z-kvm.adoc#creating-multi-arch-compute-nodes-ibm-z-kvm[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with {op-system-base} KVM]
+|&#10003;
+|
+|`x86_64`, `s390x`
+|`x86_64`, `s390x`
+
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc#creating-multi-arch-compute-nodes-ibm-power[Creating a cluster with multi-architecture compute machines on {ibm-power-name}]
+|&#10003;
+|
+|`x86_64`, `ppc64le`
+|`x86_64`, `ppc64le`
+
+|===
```

Comment:
This may sound strange, it may be worth adding a note that the Control Plane supports only a single homogeneous architecture

Reply:
Hello @prb112! I'm not sure if I know what you mean, do you mean in a cluster you can't have two control planes, where one is one arch and the other is a different one?

Reply:
@kelbrown20 @prb112 I just discussed several options to make things clearer with @werled 
We have lots of ideas but getting this merged before GA ....

How about changing the `,` to `or` in the control plane column? 
For example: 
Line 71: `x86_64` or `ppc64le`

Reply:
I like that too, thank you @prb112 and @SNiemann15